application/tests/org.openjdk.jmc.flightrecorder.controlpanel.ui.test/.classpath

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-17"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-21"/>
```

Comment:
Any specific reason why are we hard coding 21 here?

Reply:
Nope. Good catch! I'll take another pass and ensure that this is properly done for all classpath files.